docs/source/en/tasks/asr.mdx

```diff
@@ -157,17 +173,55 @@ Unlike other data collators, this specific data collator needs to apply a differ
 ...         return batch
 ```
 
-Create a batch of examples and dynamically pad them with `DataCollatorForCTCWithPadding`:
+Now create a batch of examples and dynamically pad them with `DataCollatorForCTCWithPadding`:
```

Comment:
This is potentially a bit misleading since we don't actually batch or pad examples here, but simply instantiate the data collator